src/Controller/NphOrderController.php

```diff
@@ -71,7 +71,7 @@ public function generateOrderAction(
             'stoolSamples' => $nphOrderService->getSamplesByType('stool'),
             'nailSamples' => $nphOrderService->getSamplesByType('nail'),
             'samplesOrderIds' => $nphOrderService->getSamplesWithOrderIds(),
-            'samplesStatus' => $nphOrderService->getSamplesWithStatus()
+            'samplesStatus' => $nphOrderService->getSamplesWithStatus(),
```

Comment:
Unnecessary trailing comma
```suggestion
            'samplesStatus' => $nphOrderService->getSamplesWithStatus()
```